changelog.md

```diff
@@ -10,4 +10,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - [[#3012] Add support to decimal, records and arrays types](https://github.com/ballerina-platform/ballerina-standard-library/issues/3012#issue-1274825828)
 - [[#3013] Add support for record with cyclic references](https://github.com/ballerina-platform/ballerina-standard-library/issues/3013#issue-1274864592)
-- [[#3032] Add support to ballerina map type](https://github.com/ballerina-platform/ballerina-standard-library/issues/3032#issue-1283701869)
\ No newline at end of file
+- [[#3032] Add support to ballerina map type](https://github.com/ballerina-platform/ballerina-standard-library/issues/3032#issue-1283701869)
```

Comment:
Let's point to the issue, instead of the issue description. Check all the occurrences. 
```suggestion
- [[#3032] Add support to ballerina map type](https://github.com/ballerina-platform/ballerina-standard-library/issues/3032)
```

Reply:
+1